### ISSUE TITLE
9.2.0 - Dark themes - Unsee eye in settings doesn't match color scheme

### ISSUE THREAD

**Issue by MySmilingTurban** [KhalisFoundation/sttm-desktop#1857]
Steps to reproduce
- Select any dark mode
- Open settings and disable display of any type
- Notice unsee eye icon is black, should be white to meet the color contrast

![Screenshot 2023-11-25 at 1 21 25 pm](https://github.com/KhalisFoundation/sttm-desktop/assets/51584037/4c3f40da-3922-45a7-aa31-8c1ecfee0059)
